package/boot/uboot-envtools/files/mediatek_mt7622

```diff
@@ -38,6 +38,9 @@ buffalo,wsr-2533dhp2)
 ubnt,unifi-6-lr-ubootmod)
 	ubootenv_add_uci_config "/dev/mtd2" "0x0" "0x4000" "0x10000"
 	;;
+xiaomi,redmi-router-ax6s)
+	ubootenv_add_uci_config "/dev/mtd3" "0x0" "0x10000" "0x40000"
+	;;
```

Comment:
We usually also split adding setting for uboot-envtools into a separate commit titled `uboot-envtools: ...`, but I personally don't mind it happening here.

Reply:
You need to add this line: `ubootenv_add_uci_sys_config "/dev/mtd4" "0x0" "0x1000" "0x40000"`
Details here: https://github.com/openwrt/openwrt/pull/4936

Reply:
I briefly had it in there before, but really couldn't find a use for it since the majority of the fields were in the usual uboot-env, and I already had access.

You're probably right it should go back in, as I believe you would be able to keep maintain access to devices after restoring back to stock.

Moreso for the Chinese-sold units, as these are coming out of the factory with telnet_en=0 in the bdata, whereas the "global" units are coming out with telnet_en=1, allegedly.